ios/FluentUI.Tests/DatePickerControllerTests.swift

```diff
@@ -7,10 +7,8 @@ import XCTest
 @testable import FluentUI
 
 class DatePickerControllerTests: XCTestCase {
-    static let testDateInterval: TimeInterval = 1551903381
-
-    let startDate = Date(timeIntervalSince1970: DatePickerControllerTests.testDateInterval)
-    let endDate = Date(timeIntervalSince1970: DatePickerControllerTests.testDateInterval).adding(days: 1)
+    let startDate = NSDate.now
```

Comment:
https://github.com/microsoft/fluentui-apple/actions/runs/3833281882/jobs/6524576949#step:5:305 looks like they want it to have an explicit type interface